server/models/document.py

```
@@ -0,0 +1,41 @@
+from datetime import timezone, datetime
+from sqlalchemy.orm import validates
+from . import db
+
+class Document(db.Model):
+    __tablename__ = "documents"
+    
+    id = db.Column(db.Integer, primary_key=True)
+    admin_id = db.Column(db.Integer, db.ForeignKey(users.id))
```

Ruff (0.12.2)

Undefined name `users`

(F821)